cluster/gce/gci/configure-helper.sh

```diff
@@ -279,6 +279,11 @@ function create-master-auth {
   local -r auth_dir="/etc/srv/kubernetes"
```

Comment:
Please update the comment starting on line 274 to describe how the new variable interacts with the files on the master pd.